lib/views/after_auth_screens/events/event_filter_bottomsheet.dart

```diff
@@ -41,85 +44,100 @@ Widget dropDownList(ExploreEventsViewModel model, BuildContext context) {
                       key: const Key('close'),
                       onPressed: () {
                         Navigator.pop(context);
                       },
-                      icon: const Icon(Icons.close),
+                      icon: const Icon(
+                        Icons.close,
+                      ),
                     ),
                   ],
                 ),
-                ...List.generate(filters.length, (index) {
-                  return Padding(
-                    padding: EdgeInsets.symmetric(
-                      vertical: SizeConfig.safeBlockVertical! * 2,
-                    ),
-                    child: GestureDetector(
-                      onTap: () {
-                        model.choseValueFromDropdown(
-                          filters.keys.toList()[index],
-                        );
-                        setState(() {});
-                      },
-                      child: Container(
-                        key: Key('${filters.keys.toList()[index]}test'),
-                        decoration: BoxDecoration(
-                          color:
-                              model.chosenValue == filters.keys.toList()[index]
-                                  ? Theme.of(context).colorScheme.secondary
-                                  : AppTheme.white,
-                          borderRadius:
-                              const BorderRadius.all(Radius.circular(8)),
-                        ),
-                        width: SizeConfig.screenWidth! - 60,
-                        child: Padding(
-                          padding: EdgeInsets.symmetric(
-                            horizontal: SizeConfig.safeBlockHorizontal! * 5,
-                            vertical: SizeConfig.safeBlockVertical! / 2,
+                ...List.generate(
+                  filters.length,
+                  (index) {
+                    return Padding(
+                      padding: EdgeInsets.symmetric(
+                        vertical: SizeConfig.safeBlockVertical! * 2,
+                      ),
+                      child: GestureDetector(
+                        onTap: () {
+                          model.choseValueFromDropdown(
+                            filters.keys.toList()[index],
+                          );
+                          setState(() {});
+                        },
+                        child: Container(
+                          key: Key(
+                            '${filters.keys.toList()[index]}test',
+                          ),
+                          decoration: BoxDecoration(
+                            color: model.chosenValue ==
+                                    filters.keys.toList()[index]
+                                ? Theme.of(context).colorScheme.secondary
+                                : AppTheme.white,
+                            borderRadius: const BorderRadius.all(
+                              Radius.circular(8),
```

Comment:
8 here is radius.? do you want me to make this dynamic? I mean it would look weird to make it dependent upon the width or height. Also 8px radius would never cause any overflow error. 

I can do it. But I dont think there is any need. Also in whole project we are using constant radius only.